packages/react-composites/src/composites/common/icons.tsx

```diff
@@ -181,17 +181,76 @@ export type CallCompositeIcons = Partial<
   >
 >;
 
-/* @conditional-compile-remove-from(stable): meeting-composite */
```

Comment:
This type should stay in beta-only for now.

Reply:
It's removed from export in the index.ts now -- needed used in the CallWithChatComposite file in stable build as its still built but also just not exported